wiki/StyleGoals.md

```diff
@@ -44,3 +44,36 @@ Ideally, each species should have slightly different speech patterns to indicate
+
+### Writing style
+
+Endless Sky's writing has a certain tone and voice. A few guidelines for writing missions include:
+
+* Text should be concise, descriptive, and not overly flowery. It should be functional above all else. That's not to say that Endless Sky can't have interesting or lively narration, but keep in mind that this is a game, not a novel. Please avoid walls of text when writing.
+
+* The "yes, I want this mission" choice in a conversation should usually come first in the list (so it is the default). The option to reject a mission should come last in a choice, so that players do not accidentally decline plot-important missions.
+
+* Don't imply that the player character is a certain gender in conversations. The player character's gender is deliberately left ambiguous and up to the player. 
+
+* Single option choices (i.e. a `choice` in a conversation that contains only one possible response) are heavily discouraged. The point of using `choice` is to give the player the freedom to choose how they want to respond to something. Having only one option defeats this point and effectively railroads the player into 'choosing' to do or say something that they may not want to do. If the player must do something (e.g. they're being physically forced to go somewhere), then simply state that they do so in the narration - using `choice` is unnecessary.
+
+* Generally speaking, avoid telling the player how they feel about a situation, as the player may not feel the same way. Writing such as 'You feel a warm glow when you meet this person' is not appropriate for the game. The narration's description of characters and situations should stay neutral and let the player come to their own conclusions. Similarly, please don't make major decisions for the player when writing conversations - the player should be given a choice on what to say or do in a situation.
+
+* While Endless Sky does touch on heavy subject matter (slavery, war crimes, etc.) the player's direct POV should not be overly graphic, violent, or sexual. Endless Sky is not that kind of game.
+
+* Be sure to go over the [QualityChecklist](https://github.com/endless-sky/endless-sky/wiki/QualityChecklist) when creating missions.
+
+### Punctuation and grammar
+
+Not the most thrilling part of the game, but it's important to maintain consistency so that your writing is easier to review.
+
+* Use American English for spelling and punctuation (ex. "color" instead of "colour"). This also means that text such as nicknames should be in double quotes (First "The Player" Last instead of First 'The Player' Last).
+* Put periods inside of parentheses for `choice` options or other instances where the text within parentheses is its own independent sentence: (Remain silent.) instead of (Remain silent).
```

Comment:
```suggestion
* Put periods inside of parentheses for `choice` options or other instances where the text within parentheses is its own independent sentence: `(Remain silent.)` instead of `(Remain silent).`
```